caixa/caixa_test.go

```diff
@@ -215,7 +215,7 @@ func TestTemplateRequestCaixa_NumberOfDaysAfterExpirationEqualsOne_NumberOfDaysA
 
 	request := caixaRequestStub.WithStrictRules().Build()
 	result := fmt.Sprintf("%v", flow.From("message://?source=inline", request, getRequestCaixa(), tmpl.GetFuncMaps()).GetBody())
-	assert.Contains(t, result, "<NUMERO_DIAS>1</NUMERO_DIAS>", "Falha ao encontrar o campo <NUMERO_DIAS>1<NUMERO_DIAS> no request")
+	assert.Contains(t, result, "<NUMERO_DIAS>1</NUMERO_DIAS>", "Falha ao encontrar o campo <NUMERO_DIAS>0<NUMERO_DIAS> no request")
```

Comment:
@davidsmedeiros Não deveria ter um 0 no assert tbm ?

```suggestion
	assert.Contains(t, result, "<NUMERO_DIAS>0</NUMERO_DIAS>", "Falha ao encontrar o campo <NUMERO_DIAS>0<NUMERO_DIAS> no request")
```

Reply:
@davidsmedeiros o teste continuou funcionando porque não foi alterado o Stub 
![image](https://user-images.githubusercontent.com/18493760/187459290-4f97f395-2914-4373-a398-ead28eb3ea98.png)